api/AnimationEffect.json

```diff
@@ -46,10 +46,10 @@
             "version_added": null
           },
           "safari": {
-            "version_added": null
+            "version_added": true
```

Comment:
This is disconfirmed by evaluating `AnimationEffect` in Safari 12, it's not exposed. It is in Safari Technology Preview, so maybe the next version of Safari.

The cause is probably a failure to take into account `EnabledAtRuntime=WebAnimations`:
https://github.com/WebKit/webkit/blob/071e0ad875b4db3087abd4d88efafe83fdd78fb9/Source/WebCore/animation/AnimationEffect.idl#L27